src/operator/quantization/requantize.cc

```diff
@@ -29,6 +29,26 @@
 
 namespace mxnet {
 namespace op {
+
+#if MXNET_USE_ONEDNN == 1
+void RequantizeForwardExCPU(const nnvm::NodeAttrs& attrs,
+                            const OpContext& ctx,
+                            const std::vector<NDArray>& inputs,
+                            const std::vector<OpReqType>& req,
+                            const std::vector<NDArray>& outputs) {
+  const RequantizeParam& param = nnvm::get<RequantizeParam>(attrs.parsed);
+  auto out_type                = GetQuantizeOutputType(param);
```

Comment:
Here you created a variable for out_type and above (for quantization) you did not. Maybe unify this? Also as the conditions are the same for both this ops maybe it is worth to create a SupportDNNLQuantize func with them as there is such SupportDNNL* func for most other ops?

Reply:
Thank You for this advice, I will take a look at it and try to find the best way to deal with it. Unifying is definitely nice, but I will dig further and try to create the function, as You suggested.